### PR TITLE
Fixing Enso project classpath while updating to IGV 1.20

### PR DESCRIPTION
**Pull request by JaroslavTulach**
### Pull Request Description

Classpath of projects with `module-info.java` is incorrectly computed and thus we cannot stop on breakpoints in VSCode, NetBeans nor new IGV version 1.20 that was [integrated into GraalVM repository](https://github.com/oracle/graal/pull/9645). Fixing generator of `.enso-source*` files, adjusting documentation and functionality.

### Checklist

Please ensure that the following checklist has been satisfied before submitting the PR:

- [x] The documentation has been updated, if necessary.
- [x] Screenshots/screencasts have been attached, if there are any visual changes. For interactive or animated visual changes, a screencast is preferred.
- [x] All code follows the
      [Scala](https://github.com/enso-org/enso/blob/develop/docs/style-guide/scala.md),
      [Java](https://github.com/enso-org/enso/blob/develop/docs/style-guide/java.md),
      [TypeScript](https://github.com/enso-org/enso/blob/develop/docs/style-guide/typescript.md),
      and
      [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md)
      style guides. In case you are using a language not listed above, follow the [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md) style guide.
- [x] Unit tests have been written where possible.

### CODE REVIEW

**Comment by JaroslavTulach** [comment]
There are **stop** and **re-run** buttons in the output tab now:

IGV output

**Comment by JaroslavTulach** [comment]
89545ef adds tighter integration with IGV:

Enable IGV Integration

It checks the properties IGV is using. Obtains the port it listens to and passes that to the JVM executing Enso. If the IGV isn't listening, it asks the user whether listening should be enabled.

File: tools/enso4igv/IGV.md [hunk 1]
```diff
@@ -17,9 +17,9 @@ major operating systems.
 
 ## Installation
 
-Visit [GraalVM's IGV page](https://www.graalvm.org/jdk22/tools/igv/) to read and
+Visit [GraalVM's IGV page](https://www.graalvm.org/22.1/tools/igv/) to read and
 download _IGV_. Or follow
-[this link](https://lafo.ssw.uni-linz.ac.at/pub/idealgraphvisualizer/idealgraphvisualizer-0.31-cb98bbf5fef-all.zip)
+[this link](https://lafo.ssw.uni-linz.ac.at/pub/idealgraphvisualizer/idealgraphvisualizer-1.20-ea624c6066a.zip)
```
Comment (JaroslavTulach) [inline tools/enso4igv/IGV.md:22]: @Akirathan, @4e6, @hubertp please make sure you upgrade your IGV.

File: engine/common/src/test/java/org/enso/common/EnsoSourcesTest.java [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Properties;
+import org.junit.Test;
+
+public class EnsoSourcesTest {
+  @Test
+  public void verifyEngineCommonSourcesFile() throws Exception {
+    var url = EnsoSourcesTest.class.getProtectionDomain().getCodeSource().getLocation();
+    var where = new File(url.toURI());
```
Comment (JaroslavTulach) [inline engine/common/src/test/java/org/enso/common/EnsoSourcesTest.java:18]: Massive introduction of `module-info.java` done in #10823 confuses our [VSCode & IGV support](https://github.com/enso-org/enso/blob/develop/tools/enso4igv/README.md) and breaks debugging. Writing a test to ensure the `input` property is properly computed and similar regressions are avoided next time.

Comment (JaroslavTulach) [reply engine/common/src/test/java/org/enso/common/EnsoSourcesTest.java:18]: In addition to the test, here is a visual verification. Projects are found and there are no errors among them:

no errors in Enso projects

File: tools/enso4igv/IGV.md (hunk 1 above)
Comment (JaroslavTulach) [reply tools/enso4igv/IGV.md:22]: @Akirathan, what's the best way to regenerate **all** `.enso-sources*` files? Is there a way to compile everything in our `sbt` based system? Right now I have problems with wrappers like `akka-wrapper` - Not even `clean` and `compile` works for them. At the end I did:
```bash
enso$ rm -rf *
enso$ git checkout -f
enso$ sbt
sbt:enso> buildEngineDistribution
sbt:enso> testOnly Nothing
sbt:enso> bench^C
```
However that's very tedious. Do you think there is a way to invoke all Java compilations only? Moreover just generate the `.enso-sources*` files, but don't perform any compilation (that might be done with a simple property check in `FrgaalJavaCompiler.scala`)? E.g. a command to do all Java compilations (main, test, bench, etc.) for all aggregated projects is what I am looking for.

Comment (Akirathan) [reply tools/enso4igv/IGV.md:22]: `akka-wrapper/compile` does not do anything on purpose. The "correct" command to compile this project is `akka-wrapper/exportedModule`. The thing is that it is not possible to change the default behavior of the `compile` command without breaking everything else in sbt. Therefore, the new command.

But I understand your frustration in the context of this task. Generating `.enso-sources*` files is currently done in `FrgaalJavaCompiler` as part of the `compile` command, but it seems that it is, in fact, independent of the compilation and therefore, can be moved to a completely independent task. If you are interested in this pursuit, please, create an issue for that and assign it to me.

Comment (JaroslavTulach) [reply tools/enso4igv/IGV.md:22]: I can type `sbt:enso> exportedModule` - if that is the right command to force compilation even of `akka-wrapper`.

Comment (Akirathan) [reply tools/enso4igv/IGV.md:22]: I am not sure that will work. `buildEngineDistributionNoIndex` works for sure - it invokes `exportedModule` for all the dependencies that are build into the `built-distribution/.../component` directory, including all the wrappers.

**Review by Akirathan — APPROVED** [review]

**Review by hubertp — APPROVED** [review]

**Pull request merged**